style.css

```diff
@@ -0,0 +1,138 @@
+body {
```

Comment:
- [ ] There's a gap on top and to the left that can be resolved by adding a margin of 0.

Reply:
@JuliCarracedo Can you explain this further?

Reply:
When I deployed I could see that the body had a margin that was separating your content from the screen's limit by like 4 pixels on the top and left sides. I already closed my machine so I can't really show you what it looked like.